VTEX - Logistics API.json

```diff
@@ -5003,433 +4929,307 @@
                                                             },
                                                             "deliveryPointId": {
                                                                 "type": "string",
+                                                                "description": "Delivery point ID.",
                                                                 "nullable": true
                                                             },
                                                             "point": {
                                                                 "type": "string",
+                                                                "description": "Delivery point.",
                                                                 "nullable": true
                                                             },
-                                                            "inStore": {
-                                                                "title": "InStore1",
-                                                                "required": [
-                                                                    "IsCheckedIn",
-                                                                    "StoreId"
-                                                                ],
+                                                            "inStore": {                                                                
                                                                 "type": "object",
+                                                                "description": "Relates to [VTEX Sales App](https://help.vtex.com/en/tracks/instore-getting-started-and-setting-up--zav76TFEZlAjnyBVL5tRc).",
                                                                 "properties": {
                                                                     "IsCheckedIn": {
-                                                                        "type": "boolean"
+                                                                        "type": "boolean",
+                                                                        "description": "This field is set as `true` when the order was placed via [VTEX Sales App](https://help.vtex.com/en/tracks/instore-getting-started-and-setting-up--zav76TFEZlAjnyBVL5tRc) and `false` when it was not."
                                                                     },
                                                                     "StoreId": {
-                                                                        "type": "string"
+                                                                        "type": "string",
+                                                                        "description": "ID of purchases made in the GoCommerce context.",
```

Comment:
what is gocommerce?

Reply:
It is something very old in VTEX, I thin it was an integration we had, I do not have more information, I got this one with Mourão in a breaking change of an unofficial Checkout API. But it was not a relevant field, so no more investigation was done.